t/16-schema-upgrade-html.t

```diff
@@ -147,27 +157,32 @@ $out = html_formatter_context {
 } test_request();
 
 filter_js_src($out);
+my $mode = find_application_mode($out);
 
-my @expected = split (/\n/, q{<!-- prettier-disable -->
+my @expected = split (/\n/, qq{<!-- prettier-disable -->
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
     <title></title>
     <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
-    <link rel="stylesheet" href="js/dojo/resources/dojo.css" type="text/css" />
-    <link rel="stylesheet" href="js/dijit/themes/claro/claro.css" type="text/css" />
-    <link rel="stylesheet" href="css/ledgersmb.css" type="text/css" />
-    <link rel="stylesheet" href="css/setup.css" type="text/css" />
+    <link href="js/css/claro.css" rel="stylesheet">
+    <link href="js/css/ledgersmb.css" rel="stylesheet">
+    <link href="js/css/setup.css" rel="stylesheet">
     <script>
         var dojoConfig = {
             async: 1,
             locale: "",
-            packages: [{"name":"lsmb","location":"../lsmb"}]
+            packages: [{"name":"lsmb","location":"../lsmb"}],
+            mode: "$mode"
         };
         var lsmbConfig = {
         };
     </script>
-    <script src="js/lsmb/main.js" charset="utf-8"></script>
+    } .
+    ($mode eq "production"
+        ? q{<script src="js/manifest.js"></script><script src="js/npm.dijit.js"></script><script src="js/npm.dojo.js"></script><script src="js/npm.dojo-webpack-plugin.js"></script><script src="js/gnome~gnome2~ledgersmb~ledgersmb-blue~ledgersmb-brown~ledgersmb-common~ledgersmb-purple~ledgersmb-re.js"></script><script src="js/main.js"></script>}
```

Comment:
IMO, it would be nice if these would be on separate lines for readability. Is that an option?

Reply:
Nope, generated by html webpack plugin which inline its css and js outputs

Reply:
By having it in the config, I am able to put it in the compile bundle directly without needing a bundle and allow `html-webpack-plugin` to fix the CSS & JS required, which are hashed named to allow browser caching on user installation to work properly.

Reply:
See https://github.com/ledgersmb/LedgerSMB/pull/4787/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR232 & https://github.com/ledgersmb/LedgerSMB/pull/4787/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR262 for injections points

Reply:
ok. if that's how it works, then that's how it works.